packages/discord-snippet/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: "discord-snippet"
+title: "Discord Snippets"
+description: A Collection of snippets for Discord Markdown
+version: 0.1.0
+author: Nathanel Moulin
+tags: ["utility"]
```

Comment:
You might like to add `, "discord"` as a tag, and perhaps others (see https://hub.espanso.org/search for examples) to make your package easy to find in the Hub?